packages/enhanced/README.md

```diff
@@ -1,11 +1,168 @@
-# enhanced
+# `@module-federation/enhanced`
```

Comment:
I'm not sure we should export the rspack enhancement plugin in enhanced

Reply:
If we add TypeHints and other enhanced features , i think it's necessary to add rspack enhanced